src/data_structures/arraylist.md

```diff
@@ -0,0 +1,21 @@
+
+### Arraylist
+
+The Array list in the `java.util` package acts as a normal Array with the exception of being able to easily modify its values. In an `Array` you need to resize the data structure to add/remove an item. `ArrayList` does this for you automatically so you don't need to resize the Array manually. **Elements inside an Array list are Objects** Thus `Type` is a wrapper equivalent of a type. 
+
+```java
+import java.util.ArrayList;
+
+ArrayList<Type> Name = new ArrayList<Type>(); 
+```
+
+***Functions***
+
+```java
```

Comment:
The way this book is laid out for other things, each of these methods would be their own page alongside an explanation of when you would use them and some appropriate examples